app/livechat/server/lib/Livechat.js

```diff
@@ -1097,6 +1097,25 @@ export const Livechat = {
 		return true;
 	},
 
+	getRoomMessages({ rid }) {
+		check(rid, String);
+
+		const isLivechat = Promise.await(Rooms.findByTypeInIds('l', [rid])).count();
+
+		if (!isLivechat) {
+			return [];
+		}
+
+		const ignoredMessageTypes = ['livechat_navigation_history', 'livechat_transcript_history', 'command', 'livechat-close', 'livechat-started', 'livechat_video_call'];
```

Comment:
Should we include `user joining/leaving` message types? I guess those ones are `uj` or `ul` 🤔

Reply:
If this data is not relevant, I can add them to the list

Reply:
For now, I don't think it would affect 🤔 so let's leave it with the current list